uint16 APS_Feedback
uint16 Brake_ACT_Feedback
uint8 Gear_position_Feedback
int16 Steering_angle_Feedback

uint8 Estop_Switch
uint8 AutoStandby_Switch
uint8 APM_Switch
uint8 ASM_Switch
uint8 AGM_Switch

uint8 Override_Feed

uint16 avgSpeed

uint8 lampSignal

uint8 BPS_Feed
uint8 APS_Feed

uint16 WS_FL
uint16 WS_FR
uint16 WS_RL
uint16 WS_RR